packages/plugin-image-generation/src/index.ts

```diff
@@ -116,9 +126,27 @@ const imageGeneration: Action = {
         const images = await generateImage(
             {
                 prompt: imagePrompt,
-                width: 1024,
-                height: 1024,
-                count: 1,
+                width: options.width || 1024,
+                height: options.height || 1024,
+                ...(options.count !== undefined
+                    ? { count: options.count || 1 }
```

Comment:
!= null for these as we probably want the {} on null and undefined.

Reply:
Will change to `...(options.count != null ? { count: options.count || 1 } : {}),`